.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: node_js
+addons:
+    code_climate:
+        repo_token: tanxchangeme!46ecd492907f8224b0ae64890d4befb4bbd9ebbeba55b5acb38
```

Comment:
you need to access to code climate and as you are the owner you should read the token needed for openpgpjs/openpgpjs.

you find it under openpgpjs/openpgpjs section inside the code coverage subsection.